### ISSUE TITLE
Rule proposal: forced disabling reason

### ISSUE THREAD

**Issue by dartess** [mysticatea/eslint-plugin-eslint-comments#47]
_copy from the main repository (https://github.com/eslint/eslint/issues/13338)_

**Please describe what the rule should do:**

oblige the developer to explain the reason for disconnecting the linter

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Enforces code style (layout)
[x] Other (please specify:) —  reduce the number of thoughtless shutdowns of the linter; Improve code reading speed by other developers.

**Provide 2-3 code examples that this rule will warn about:**

```js
// fail
// eslint-disable-next-line no-new
new Swiper(slider);

// ok
// eslint-disable-next-line no-new -- saving the instance is not necessary, since subsequent control of the slider is not needed
new Swiper(slider);
```

**Why should this rule be included in ESLint (instead of a plugin)?**

I'm not sure, but it seems to me that this will reduce the number of incorrect, incomprehensible and lazy shutdowns of the linter

**Are you willing to submit a pull request to implement this rule?**

I doubt my competence, this rule does not look banal.

**Comment by ota-meshi** [comment]
Hi.

I think your proposed rule is already implemented in the master. #44 
However, it has not been released yet.

**Comment by dartess** [comment]
Yes that's it.
Thank! 🖤
